sensors/nxt_i2c_sensor_core.c

```diff
@@ -427,7 +471,7 @@ struct i2c_driver nxt_i2c_sensor_driver = {
 	.class		= I2C_CLASS_LEGOEV3,
 	.detect		= nxt_i2c_sensor_detect,
 	.address_list	= I2C_ADDRS(0x01, 0x02, 0x03, 0x08, 0x0A, 0x0c, 0x11, 0x18,
-				    0x50, 0x51, 0x52, 0x58),
+				    0x4C, 0x50, 0x51, 0x52, 0x58),
```

Comment:
I'm not sure which would be more consistent, lowercase or upper case. We seem to have both. Lowercase is more common though, I think.